src/lib/components/Code.svelte

```diff
@@ -19,6 +19,10 @@
   }
 </script>
 
+<svelte:head>
```

Comment:
to fix (styles not always were applied):
![css_bug](https://github.com/user-attachments/assets/eb83b98c-3dd6-4f24-b6c5-823be3651ec1)